demos/browser/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Amazon Chime SDK JavaScript Demos",
   "scripts": {
     "deps": "cd ../.. && npm run build",
-    "build:fast": "tsc && webpack --config ./webpack.config.js",
-    "build": "npm run deps && npm install && npm run build:fast",
+    "build:fast": "node script/webpack-wrapper.js",
+    "build": "npm run deps && script/conditional-install && npm run build:fast",
     "start:fast": "npm run build:fast && node serve.js",
     "start:hot": "tsc && webpack-cli serve --config ./webpack.config.js",
     "start": "npm run deps && npm install && npm install aws-sdk && npm run start:fast"
```

Comment:
Don't we need to use script/conditional-install here too?
Also can we remove npm install aws-sdk? Don't know why we need to install aws-sdk again everytime.

Reply:
I agree :)